combineRooms/combineRooms.py

```diff
@@ -0,0 +1,320 @@
+import discord
+from redbot.core import Config
+from redbot.core import commands
+from redbot.core import checks
+
+# TODO:
+# - custom combines_info message
+# - set league acronym (default = RSC)
+defaults = {"players_per_room": 6, "room_capacity": 10, "combines_category": None}
+
+
+class CombineRooms(commands.Cog):
+    def __init__(self, bot):
+        self.config = Config.get_conf(self, identifier=1234567892, force_registration=True)
+        self.config.register_guild(**defaults)
+        self.team_manager_cog = bot.get_cog("TeamManager")
+        self._combines_category_name = "Combine Rooms"
+
+    @commands.command(aliases=["startcombines", "stopcombines"])
+    @commands.guild_only()
+    @checks.admin_or_permissions(manage_guild=True)
+    async def combines(self, ctx, action=None):
+        """
+        Creates rooms for combines, or tears them down depending on the action parameter. If no parameter is given, it will behave as a switch.
+        
+        Examples:
+        [p]combines
+        [p]combines start
+        [p]combines stop
+        """
+        if action in ["start", "create"]:
+            done = await self._start_combines(ctx)
+        elif action in ["stop", "teardown", "end"]:
+            done = await self._stop_combines(ctx)
+        else:
+            combines_ongoing = await self._combines_category(ctx.guild)
+            if combines_ongoing:
+                done = await self._stop_combines(ctx)
+            else:
+                done = await self._start_combines(ctx)
+        
+        if done:
+            await ctx.send("Done")
+        return
+    
+    @commands.command(aliases=["sppr"])
+    @commands.guild_only()
+    @checks.admin_or_permissions(manage_guild=True)
+    async def set_players_per_room(self, ctx, size: int):
+        if size < 2:
+            await ctx.send("No.")
```

Comment:
Is this meant to be an error message saying that the size can't be less than 2? If so, make sure the message sent reflects that rather than just "No."

Reply:
Will do, I forgot about this line. I just wanted to test behavior and hadn't come back to writing a better error response. But yes, having no more than 1 player per room would be pointless.

I'll make this print differently.

Reply:
Done.